src/scss/_font_pairing_casual.scss

```diff
@@ -0,0 +1,39 @@
+html.emc-font-pairing--casual {
+  font-family: $font-face-body-casual;
+}
+
+.emc-font-pairing--casual h1, .emc-font-pairing--casual.emc-h1,
+.emc-font-pairing--casual h2, .emc-font-pairing--casual.emc-h2,
+.emc-font-pairing--casual h3, .emc-font-pairing--casual.emc-h3,
+.emc-font-pairing--casual h4, .emc-font-pairing--casual.emc-h4,
+.emc-font-pairing--casual h5, .emc-font-pairing--casual.emc-h5,
+.emc-font-pairing--casual h6, .emc-font-pairing--casual.emc-h6,
+.emc-font-pairing--casual.emc-page-title,
+.emc-font-pairing--casual.emc-display-heading,
+.emc-font-pairing--casual.emc-block-heading,
+.emc-font-pairing--casual.emc-card__heading,
+.emc-font-pairing--casual.emc-news-card  {
+  font-family: $font-face-heading-casual;
+}
+
+.emc-font-pairing--casual.footer.emc-footer h1,
+.emc-font-pairing--casual.footer.emc-footer h2,
+.emc-font-pairing--casual.footer.emc-footer h3,
+.emc-font-pairing--casual.footer.emc-footer h4,
+.emc-font-pairing--casual.footer.emc-footer h5,
+.emc-font-pairing--casual.footer.emc-footer h6,
+.emc-font-pairing--casual.emc-card__footer,
+.emc-font-pairing--casual.emc-pager,
+.emc-font-pairing--casual.emc-btn,
+.emc-font-pairing--casual.emc-breadcrumb,
+.emc-font-pairing--casual.emc-modal {
+  font-family: $font-face-body-casual;
+}
+
+.emc-font-pairing--casual.emc-event-card__title a {
+  font-family: $font-face-heading-casual;
+}
+
+.emc-font-pairing--casual.a.emc-tab-link {
+  font-family: $font-face-body-casual;
+}
```

Comment:
The headings (h1-h6) are correct, but I think that all of the other selectors also need to be adjusted.

What you have currently is saying that for emc-h1 (through emc-h6), the casual fonts will only be applied if the headings also have an explicit emc-font-pairing--casual class specified on the heading also.  The emc-hn classes should behave the same as h1-h6 and should be something like this:

.emc-font-pairing--casual emc-h3

The same goes for all of the other cases where you have the classes chained together.  For example:

.emc-font-pairing--casual.footer.emc-footer h1

...should be...

.emc-font-pairing--casual footer.emc-footer h1

Reply:
You are right. I should've realized that somehow when I tested it though the fonts were being switched correctly. Will modify and test again. Thanks!